nodes/FormX/actions/document/commonProperties.ts

```diff
@@ -1,4 +1,28 @@
-import { INodeProperties } from 'n8n-workflow';
+import {
+	IExecuteSingleFunctions,
+	IHttpRequestOptions,
+	INodeProperties,
+	NodeOperationError,
+} from 'n8n-workflow';
+
+export async function getImageBinaryData(
+	this: IExecuteSingleFunctions,
+	requestOptions: IHttpRequestOptions,
+): Promise<IHttpRequestOptions> {
+	try {
+		const binaryDataField = this.getNodeParameter('binaryDataField');
+
+		let dataBuffer: Buffer | null = null;
+		if (binaryDataField && this.helpers.assertBinaryData(binaryDataField)) {
+			dataBuffer = await this.helpers.getBinaryDataBuffer(binaryDataField);
+		}
+		requestOptions.body = dataBuffer;
+
+		return requestOptions;
+	} catch (err) {
+		throw new NodeOperationError(this.getNode(), err as Error);
```

Comment:
What errors would user see in UI?

Reply:
Actually, I don't see big difference wrapping `NodeOperationError` or not

But seems wrapping it is a standard from what I see in other nodes. (Can't find relevant doc explaining it)
I guess it's for some n8n server internal tracking or sth?

Reply:
If you mean the error of this function, here is the screenshot. I think it's understandable

<img width="1710" alt="image" src="https://github.com/FormX-ai/n8n-nodes-formx/assets/63200569/f1098b46-01c4-4937-9288-2163e143856c">

Reply:
Hmm I wonder what about other errors, e.g. the field is present but fail to grab the buffer. But I guess for now this suffice. Please keep this in mind first, thanks!